package.json

```diff
@@ -42,6 +42,8 @@
     "eslint-plugin-prettier": "3.1.0",
     "eslint-plugin-promise": "4.2.1",
     "git-revision-webpack-plugin": "3.0.3",
+    "husky": "3.0.2",
```

Comment:
Cool. Does this mean we don't need the global install of `pre-commit` any longer?

Reply:
Yup!